### ISSUE TITLE
Termite Plot cropping words

### ISSUE THREAD

**Issue by radiantone** [chartbeat-labs/textacy#213]
The termite plot in 0.6.1 is cutting off the words on the y-axis

## Expected Behavior
For all the Y labels to be visible

## Current Behavior
Y labels are being cropped

## Steps to Reproduce (for bugs)
Code below

## Context
Python 2.7, textacy 0.6.1, Zeppelin 0.7

## Your Environment
Red Hat

```
import numpy as np
from textacy.viz.termite import draw_termite_plot

mat = np.array([[5, 4, 3, 2, 1],
                [4, 3, 2, 1, 0],
                [3, 2, 1, 0, 5],
                [2, 1, 0, 5, 4],
                [1, 0, 5, 4, 3]])
                
draw_termite_plot(mat,['A','B','C','D','E'],['word 1','word 2','word 3','word 4','word 5'],highlight_cols=range(0,3))
```
![termite](https://user-images.githubusercontent.com/25509097/48157685-49764d00-e29e-11e8-87b3-5f483758d0ea.png)

**Comment by bdewilde** [comment]
Hi @radiantone , thanks for your patience and apologies for the delay, I've not had bandwidth for `textacy` dev work for a few months.

I've always had trouble debugging this sort of issue — `matplotlib` can be finicky! Unfortunately, I'm unable to reproduce it in my setup using either `matplotlib` 2.2 or 3.0:

<img width="637" alt="screen shot 2018-12-02 at 1 42 12 pm" src="https://user-images.githubusercontent.com/2514535/49344059-183c3300-f638-11e8-88b3-ce8cbc8a9d84.png">

Could you add `constrained_layout=True` to [this line](https://github.com/chartbeat-labs/textacy/blob/master/textacy/viz/termite.py#L111) where the plot's figure and axes are being initialized, and let me know if that fixes it?

**Comment by bdewilde** [comment]
Hi @radiantone , you've probably moved on from this, but just in case: today I merged in a PR (#248) that included changes to help address this, and there's a brand new v0.8.0 release with them included. I'm going to close this issue out, but please reopen if you're still stuck and unable to make it work.
